src/Compilers/CSharp/Portable/Symbols/Source/SourceDelegateMethodSymbol.cs

```diff
@@ -303,26 +303,24 @@ internal override LexicalSortKey GetLexicalSortKey()
 
             internal override void AfterAddingTypeMembersChecks(ConversionsBase conversions, DiagnosticBag diagnostics)
             {
-                Location getReturnTypeLocation()
```

Comment:
>getReturnTypeLocation() [](start = 25, length = 23)

Not related to this PR: Any idea why we were doing those gymnastics before? Seems really strange.

Reply:
I added the local functions in #22820 to allow the locations to be calculated late without duplicating code in each place that needed a location.

---
In reply to: [205842705](https://github.com/dotnet/roslyn/pull/28804#discussion_r205842705) [](ancestors = 205842705)